dss-pades/src/main/java/eu/europa/esig/dss/pdf/pdfbox/PdfBoxSignatureService.java

```diff
@@ -178,9 +178,22 @@ protected void fillImageParameters(final PDDocument doc, final SignatureImagePar
 				PDVisibleSignDesigner visibleSig = new PDVisibleSignDesigner(doc, is, signatureImageParameters.getPage());
 				visibleSig.xAxis(signatureImageParameters.getxAxis());
 				visibleSig.yAxis(signatureImageParameters.getyAxis());
-				visibleSig.width(ires.toXPoint(visibleSig.getWidth()));
-				visibleSig.height(ires.toYPoint(visibleSig.getHeight()));
-				visibleSig.zoom(signatureImageParameters.getZoom() - 100); // pdfbox is 0 based
+                                
+                                
+                                
+                                if(signatureImageParameters.getWidth()== 0 && signatureImageParameters.getHeight()== 0){
```

Comment:
Hello Pierrick,
you're right of course, I wanted to test something with this change and instead of reverting it back unintentionally commited it. I will revert the change again.
Kind regards,
Pierrick